docs/reference/ilm/policy-definitions.asciidoc

```diff
@@ -204,6 +204,21 @@ PUT _ilm/policy/my_policy
 }
 --------------------------------------------------
 
+NOTE: To add attributes `box_type` with values of `hot` and `warm` add lines to `elasticsearch.yaml`:
```

Comment:
I don't think this needs to be a note, just a regular paragraph. A little more explanation would be helpful. I'd show a single alternative and show the example inline, since it's not the main focus here:

This depends on `hot` and `warm` attributes specified in the node configurations. For example, to designate a node as a _hot_ node, set `node.attr.data: hot` in `elasticsearch.yml`. For more information, see <<index-allocation-filters>>.

Reply:
Done, thanks!!
![image](https://user-images.githubusercontent.com/25182304/78956232-a4256100-7aaf-11ea-82c4-afbf22c68a41.png)